ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -234,8 +233,20 @@ def get_bkg_dict(self, task_config: Dict[str, Any]) -> Dict[str, List[str]]:
             # accumulate directory list for member restart files
             dirlist.append(os.path.join(task_config.DATA, 'bkg', memchar))
 
-            # get FV3 RESTART files, this will be a lot simpler when using history files
-            rst_dir = os.path.join(task_config.comin_ges_atmens, memchar, 'atmos/RESTART')
+            # get FV3 restart files, this will be a lot simpler when using history files
+            template = task_config.COM_ATMOS_RESTART_TMPL
+            tmpl_dict = {
+                'ROTDIR': task_config.ROTDIR,
+                'RUN': self.runtime_config.RUN,
+                'YMD': to_YMD(task_config.previous_cycle),
+                'HH': task_config.previous_cycle.strftime('%H')
+            }
+
+            # get FV3 restart files, this will be a lot simpler when using history files
+            tmpl_dict['MEMDIR'] = memchar
+            rst_dir = Template.substitute_structure(template, TemplateConstants.DOLLAR_CURLY_BRACE, tmpl_dict.get)
+            os.makedirs(rst_dir, mode=0o775, exist_ok=True)
```

Comment:
While this works, the function here is to return a list of model background files to copy.
Now this function is also tasked w/ creating member directories.

We should move this to `analysis.py` and into its own method.

Reply:
I will begin work on this request later tonight or tomorrow.

Reply:
Sorry Russ, this is on me for saying you should just add it here instead of suggesting a new method to create directories.

Reply:
Requested change committed at [`da20985`](https://github.com/RussTreadon-NOAA/global-workflow/commit/da20985506a351ff6b13201f261bde5df12e99d3).